frontend/src/components/WofButton.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <button :class="`button button--${type}`" :style="`font-size: ${this.size}rem;`" @click="onClick()">
+    <wof-icon v-if="this.icon" :icon="this.icon" :size="this.size" :class="this.getIconCss" />
```

Comment:
You don't need to use word 'this' in template

Reply:
I like the verbosity of it. Unless it's against a recommendation, I would like to keep it.

Reply:
Ok keep it for now